src/vaadin-number-field.html

```diff
@@ -201,6 +212,17 @@
           }
           return super.checkValidity();
         }
+
+        _onInput(e) {
```

Comment:
Chrome allows me to type "eee" while Safari doesn't.

Reply:
What Chrome version are you using? I'm using v71 and I can't type (or paste) "e".

Reply:
Works for me now also